documentation/existing.md

```diff
@@ -94,11 +100,65 @@ In some cases, however, you may still want to customize the privileges for vSphe
+        user = "vCenter username for cloud provider"
+        password = "password"        
+        port = "443" #Optional
+        insecure-flag = "1" #set to 1 if the vCenter uses a self-signed cert
+        datacenters = "list of datacenters where Kubernetes node VMs are present"
+        
+[VirtualCenter "1.2.3.4"]
+        # Override specific properties for this Virtual Center.
+        user = "vCenter username for cloud provider"
+        password = "password"
+        # port, insecure-flag, datacenters will be used from Global section.
+        
+[VirtualCenter "1.2.3.5"]
+        # Override specific properties for this Virtual Center.
+        port = "448"
+        insecure-flag = "0"
+        # user, password, datacenters will be used from Global section.
+        
+[Workspace]
+        # Specify properties which will be used for various vSphere cloud provider functionality.
+        # e.g. Dynamic provisioing, Storage Profile Based Volume provisioning etc.
+        server = "IP/FQDN for vCenter"
+        datacenter = "Datacenter name"
+        folder = "vCenter VM folder path in which dummy VMs should be created."
+        default-datastore = "Datastore name" #Datastore to use for provisioning volumes using storage classes/dynamic provisioning
+        resourcepool-path = "Resource pool path" # Used fordummy VM creation. Optional
+
+[Disk]
+        scsicontrollertype = pvscsi
+
+[Network]
+        public-network = "Network Name to be used"
+```
+
+Below is summary of supported parameters in the `vsphere.conf` file for kubernetes version 1.9.x
+
+* ```user``` is the vCenter username for vSphere Cloud Provider.
+* ```password``` is the password for vCenter user specified with `user`.
+* ```port``` is the vCenter Server Port. Default is 443 if not specified.
+* ```insecure-flag``` is set to 1 if vCenter used a self-signed certificate.
+* ```server``` is the vCenter Server IP or FQDN
+* ```datacenters``` is the names of the datacenters on which Node VMs are deployed.
+* ```default-datastore``` is the default datastore to use for provisioning volumes using storage classes/dynamic provisioning.
+* ```resourcepool-path``` is the path to resource pool where dummy VMs for Storage Profile Based volume provisioning should be created. It is optional parameter.
+* ```folder``` VM Folder path where Node VMs can be present.
 
-Note: Cloud config template can be found [here](https://github.com/kubernetes/kubernetes-anywhere/blob/master/phase1/vsphere/vsphere.conf).
+#### For kubernetes version 1.8.x or below
 
-This config file needs to be placed in the shared directory which should be accessible from kubelet container, controller-manager pod, and API server pod.
+vSphere cloud provider config file needs to be placed in the shared directory which should be accessible from kubelet container, controller-manager pod, and API server pod.
```

Comment:
I would suggest removing pod and container. Since, it is not necessary that all these services are in running in pod or container for instance in some deployment cases this can be running using systemd.

Reply:
makes sense.